lib/activerecord-bitemporal.rb

```diff
@@ -123,6 +124,7 @@ def bitemporalize(
     extend ClassMethods
     include InstanceMethods
     include ActiveRecord::Bitemporal::Scope
+    include ActiveRecord::Bitemporal::Callbacks
```

Comment:
If it follows the same structure as upstream, it may be better to be included in `InstanceMethods`.

In the upstream, `Persistence` and `Callbacks` are included in the `Base`.
https://github.com/rails/rails/blob/v7.0.4.2/activerecord/lib/active_record/base.rb#L300
https://github.com/rails/rails/blob/v7.0.4.2/activerecord/lib/active_record/base.rb#L315

Reply:
`Callbacks` module must be included by `ActiveModel` or `ActiveRecord` class because this module calls `define_callback` method when included.
`NoMethodError` will occur if `InstanceMthods` module include `Callbacks` module.

Reply:
Fair enough. Nested `included` callbacks don't work...